gemini-2/live_api_starter.py

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 
 # To install the dependencies for this script, run:
-#  pip install google-genai opencv-python pyaudio pillow
+#  pip install google-genai opencv-python pyaudio pillow mss
 # And to run this script, ensure the GOOGLE_API_KEY environment
 # variable is set to the key you obtained from Google AI Studio.
```

Comment:
I think we should also add a line on how to enable screensharing vs. camera

Reply:
+1, added as a second commit https://github.com/google-gemini/cookbook/pull/359/commits/dacb16b34ad50c06df847e8eead68ee4b54e54ea